CRM/Appearancemodifier/Form/Event.php

```diff
@@ -176,4 +134,47 @@ private function getEvent(int $id): array
         }
         return $event->first();
     }
+    /*
+     * This function gathers the consent custom fields that
+     * are present in this petition form.
+     */
+    protected function consentActivityCustomFields(): void
+    {
+        // gather the custom fields from the service.
+        $consentActivityConfig = new CRM_Consentactivity_Config('consentactivity');
+        $consentActivityConfig->load();
+        $config = $consentActivityConfig->get();
+        if (array_key_exists('custom-field-map', $config)) {
+            $map = $config['custom-field-map'];
+            $labels = CRM_Consentactivity_Service::customCheckboxFields();
+            $uFJoins = \Civi\Api4\UFJoin::get()
+                ->addSelect('uf_group_id')
+                ->addWhere('module', '=', 'CiviEvent')
+                ->addWhere('entity_table', '=', 'civicrm_event')
+                ->addWhere('entity_id', '=', $this->event['id'])
+                ->setLimit(25)
```

Comment:
Why is this limit here?

Reply:
You can attach multiple contact profiles to a given event. I believe 25 is much more than the number of profiles that a content manager wants to attach to an event.

Reply:
So this limit doesn't have much reason

Reply:
Do you want me to remove it then?

Reply:
Yes, I think that could go.
I mean probably there won't be more than 25 number of profiles, but why limit ourselves, as I see it doesn't have any advantage.
If it has it's OK, otherwise it just could be a strange bug in the future, if for some reason we add so many profiles...

Reply:
it has been removed.